apps/music/js/ui/views/player_view.js

```diff
@@ -228,19 +230,17 @@ var PlayerView = {
         this.album.classList.remove('hidden-cover-share');
       }
     } else {
-      var titleBar = document.getElementById('title-text');
-
-      titleBar.textContent =
-        metadata.title || navigator.mozL10n.get('unknownTitle');
-      titleBar.dataset.l10nId = metadata.title ? '' : 'unknownTitle';
+      TitleBar.changeTitleText(
```

Comment:
@hfiguiere I think I was wrong about the `TitleBar.changeTitleText()`, because the else case is used for the open activity, the TitleBar object will not be initialized so that's why we directly get the titleBar element then set the textContent, let's roll back to what you had in last commit or the open activity will be broken, sorry about it!